frontend/src/scenes/notebooks/Nodes/NotebookNodeRecording.tsx

```diff
@@ -35,12 +45,44 @@ const Component = (props: NotebookNodeViewProps<NotebookNodeRecordingAttributes>
     const { sessionPlayerMetaData } = useValues(sessionRecordingDataLogic(recordingLogicProps))
     const { loadRecordingMeta } = useActions(sessionRecordingDataLogic(recordingLogicProps))
     const { expanded } = useValues(notebookNodeLogic)
+    const { setActions, insertAfter, insertReplayCommentByTimestamp } = useActions(notebookNodeLogic)
 
     useEffect(() => {
         loadRecordingMeta()
     }, [])
     // TODO Only load data when in view...
 
+    useEffect(() => {
+        const person = sessionPlayerMetaData?.person
+        setActions(
+            [
+                {
+                    text: 'Comment',
+                    icon: <IconComment />,
+                    onClick: () => {
+                        const time = getCurrentPlayerTime(recordingLogicProps) * 1000
+
+                        insertReplayCommentByTimestamp(time, id)
+                    },
+                },
+                person
+                    ? {
+                          text: `View ${asDisplay(person)}`,
+                          icon: <IconPerson />,
+                          onClick: () => {
+                              insertAfter({
+                                  type: NotebookNodeType.Person,
+                                  attrs: {
+                                      id: String(person.distinct_ids[0]),
+                                  },
+                              })
+                          },
+                      }
+                    : undefined,
+            ].filter(Boolean) as NotebookNodeAction[]
```

Comment:
There's a few different examples of conditionally setting examples in different files. Wondering if we should pick one and try sticking to it. No preference but in my mind we should go with one of 

```
const actions = []

if (conditionIsMet) {
  actions.push(firstAction)
}

setActions(actions)
```

```
setActions([
  actionOrUndefined
])

// let setActions do the `.filter(Boolean) as NotebookNodeAction[]` so that it's not a concern for the consumer
```

```
setActions([{
      text: 'Action name',
      visible:  conditionIsMet,
      onClick: () => {...}
}])

// setActions filters out the actions that are not visible
```

Reply:
That's a great point. Do the filtering in the function simplifies a lot of things